proxy-router/cmd/main.go

```diff
@@ -274,10 +274,11 @@ func start() error {
 		log.Warnf("chat context storage is disabled")
 		chatStorage = proxyapi.NewNoOpChatStorage()
 	}
+	validator := system.NewValidator()
```

Comment:
Not clear validator of what? suggest `ethConnectionValidator` or something more specific to convey meaning that it actually connects, not validetes url string.